modules/develop/pages/data-transforms/run-transforms.adoc

```diff
@@ -0,0 +1,254 @@
+= Run Data Transforms
+:description: Learn how to build and deploy WebAssembly data transforms in Redpanda.
+:page-aliases: reference:rpk/rpk-wasm/rpk-wasm.adoc, reference:rpk/rpk-wasm.adoc, reference:rpk/rpk-wasm/rpk-wasm-deploy.adoc, reference:rpk/rpk-wasm/rpk-wasm-generate.adoc, reference:rpk/rpk-wasm/rpk-wasm-remove.adoc, data-management:data-transform.adoc, labs:data-transform/index.adoc
+:latest-data-transforms-version: 0.0.0-20230830git604fcce
+
+Data transforms let you run common data streaming tasks, like filtering, scrubbing, and transcoding, within Redpanda. For example, you may have consumers that require you to remove credit card numbers or convert JSON to Avro. Data transforms can also interact with the Redpanda Schema Registry to work with encoded data types. 
```

Comment:
@rockwotj do you mean 'redact credit card numbers'?

Reply:
switched to redact